openshift_metrics/utils.py

```diff
@@ -227,17 +188,11 @@ def write_metrics_by_namespace(condensed_metrics_dict, file_name, report_month):
 
 def write_metrics_by_pod(metrics_dict, file_name):
```

Comment:
A tiny nitpick, but I would have liked this renamed to `condensed_metrics_dict` for symmetry with `write_metrics_by_namespace()` and rest of the code.

Reply:
sure, I have changed that.